platform/database_common/schema.go

```diff
@@ -211,44 +212,38 @@ func (t MultiValueType) GetColumn(name string) *Column {
 }
 
 func NewBaseType(clickHouseTypeName string) BaseType {
-	var GoType = ResolveType(clickHouseTypeName)
+	// TODO: currently, NewBaseType is only used in tests or create table or insert, not in Doris's code, so the ClickHouse schema is used here.
+	var r TypeResolver = &clickhouse.ClickhouseTypeResolver{}
+	var GoType = r.ResolveType(clickHouseTypeName)
 	if GoType == nil {
 		// default, probably good for dates, etc.
 		GoType = reflect.TypeOf("")
 	}
 	return BaseType{Name: clickHouseTypeName, GoType: GoType}
 }
 
-// this is catch all type for all types we do not exlicitly support
-type UnknownType struct{}
-
-func ResolveType(clickHouseTypeName string) reflect.Type {
-	switch clickHouseTypeName {
-	case "String", "LowCardinality(String)", "UUID", "FixedString":
-		return reflect.TypeOf("")
-	case "DateTime64", "DateTime", "Date", "DateTime64(3)":
-		return reflect.TypeOf(time.Time{})
-	case "UInt8", "UInt16", "UInt32", "UInt64":
-		return reflect.TypeOf(uint64(0))
-	case "Int8", "Int16", "Int32":
-		return reflect.TypeOf(int32(0))
-	case "Int64":
-		return reflect.TypeOf(int64(0))
-	case "Float32", "Float64":
-		return reflect.TypeOf(float64(0))
-	case "Point":
-		return reflect.TypeOf(clickhouse.Point{})
-	case "Bool":
-		return reflect.TypeOf(true)
-	case "JSON":
-		return reflect.TypeOf(map[string]interface{}{})
-	case "Map(String, Nullable(String))", "Map(String, String)", "Map(LowCardinality(String), String)", "Map(LowCardinality(String), Nullable(String))":
-		return reflect.TypeOf(map[string]string{})
-	case "Unknown":
-		return reflect.TypeOf(UnknownType{})
+func NewBaseTypeWithInstanceName(typeName string, instanceName string) BaseType {
```

Comment:
I think it would be better to use some const instead of string as `instanceName`